ml-agents-envs/mlagents/envs/sampler_class.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+from typing import *
+from functools import *
+from collections import OrderedDict
+from abc import ABC, abstractmethod
+
+from .exception import SamplerException
+
+
+class Sampler(ABC):
+    @abstractmethod
+    def sample_parameter(self) -> float:
+        pass
+
+
+class UniformSampler(Sampler):
+    """
+    Uniformly draws a single sample in the range [min_value, max_value).
+    """
+
+    def __init__(
```

Comment:
You need a doc string on the init methods so we know what the arguments correspond to